src/Solnet.Rpc/Builders/TransactionBuilder.cs

```diff
@@ -53,13 +54,14 @@ private IEnumerable<Account> GetSigners()
             
             foreach (TransactionInstruction instruction in _messageBuilder.Instructions)
             {
-                signers.AddRange(from accountMeta in instruction.Keys where accountMeta.Signer && !signers.Contains(accountMeta.Account) select accountMeta.Account);
+                signers.AddRange(from accountMeta in instruction.Keys where accountMeta.IsSigner && !signers.Contains(accountMeta.Account) select accountMeta.Account);

```

Comment:
commented code => delete

Reply:
Reason: we don't need commented code for history purposes when the code is already versioned.